src/Api.php

```diff
@@ -452,6 +454,85 @@ public function hasJoinedServer(string $username, string $serverId): Response\Pr
         return new Response\ProfileResponse($body['id'], $body['name'], $body['properties']);
     }
 
+    /**
+     * @param string $accessToken
+     * @throws GuzzleException
+     *
+     * @url https://wiki.vg/Mojang_API#Check_if_security_questions_are_needed
+     */
+    public function isSecurityQuestionsNeeded(string $accessToken): void {
+        $uri = new Uri('https://api.mojang.com/user/security/location');
+        $request = new Request('GET', $uri, ['Authorization' => 'Bearer ' . $accessToken]);
+        $response = $this->getClient()->send($request);
+        $rawBody = $response->getBody()->getContents();
+        if (!empty($rawBody)) {
+            $body = $this->decode($rawBody);
+            throw new Exception\OperationException($body['errorMessage'], $request, $response);
+        }
+    }
+
+    /**
+     * @param string $accessToken
+     * @return array
+     * @throws GuzzleException
+     *
+     * @url https://wiki.vg/Mojang_API#Get_list_of_questions
+     */
+    public function questions(string $accessToken): array {
+        $uri = new Uri('https://api.mojang.com/user/security/challenges');
+        $request = new Request('GET', $uri, ['Authorization' => 'Bearer ' . $accessToken]);
+        $response = $this->getClient()->send($request);
+        $rawBody = $response->getBody()->getContents();
+        if (empty($rawBody)) {
+            throw new Exception\NoContentException($request, $response);
+        }
+
+        $result = [];
+        $body = $this->decode($rawBody);
+        foreach ($body as $question) {
+            $result[] = [
+                'answer' => new AnswerResponse($question['answer']['id']),
+                'question' => new QuestionResponse($question['question']['id'], $question['question']['question']),
+            ];
+        }
+
+        return $result;
+    }
+
+    /**
+     * @param string $accessToken
+     * @param array $answers
+     * @throws GuzzleException
+     *
+     * @url https://wiki.vg/Mojang_API#Send_back_the_answers
+     */
+    public function answer(string $accessToken, array $answers): void {
+        $uri = new Uri('https://api.mojang.com/user/security/location');
+        $request = new Request('POST', $uri, ['Authorization' => 'Bearer ' . $accessToken], json_encode($answers));
+        $response = $this->getClient()->send($request);
+        $rawBody = $response->getBody()->getContents();
+        if (!empty($rawBody)) {
+            $body = $this->decode($rawBody);
+            throw new Exception\OperationException($body['errorMessage'], $request, $response);
+        }
+    }
+
+    /**
+     * @param array $metricKeys
+     * @return Response\StatisticsResponse
+     * @throws GuzzleException
+     *
+     * @url https://wiki.vg/Mojang_API#Statistics
+     */
+    public function statistics(array $metricKeys) {
+        $response = $this->getClient()->request('POST', 'https://api.mojang.com/orders/statistics', [
+            'json' => $metricKeys,
```

Comment:
There is no need to let end-users to specify the full request structure.
```suggestion
            'json' => [
                'metricKeys' => $metricKeys,
            ],
```